operations/iscsi_sbps/iscsi_sbps.md

```diff
@@ -31,6 +32,9 @@ In CSM 1.6, both DVS and SBPS will co-exist and DVS is planned to deprecate in C
 * Aligns with future plans for similar functionality in next generation systems management solutions
 * Easy to deploy and manage
 
+**Note:** Using HSN for boot content projection is recommended and use NMN for any debugging purposes.
+In the case of HSN is not configured, then usage of NMN is recommended if it meets the bandwidth requirements.
```

Comment:
In the case that the HSN is not configured, use the NMN if it meets the bandwidth requirements.

Reply:
Done